associations/has_one_association.go

```diff
@@ -33,12 +36,13 @@ func hasOneAssociationBuilder(p associationParams) (Association, error) {
 
 	fval := p.modelValue.FieldByName(p.field.Name)
 	return &hasOneAssociation{
-		owner:      p.model,
-		ownedModel: fval,
-		ownedType:  fval.Type(),
-		ownerID:    ownerID.Interface(),
-		ownerName:  p.modelType.Name(),
-		fkID:       p.popTags.Find("fk_id").Value,
+		owner:          p.model,
+		ownedTableName: p.popTags.Find("has_one").Value + "s",
```

Comment:
Are you sure the table name will always finish with a "s"?

Reply:
I should run it through flect instead... I'll do that.